lib/python/vdm/server/HTTPListener.py

```diff
@@ -709,15 +631,15 @@ def allowed_file(filename):
 
 def get_servers_from_database_id(database_id):
     servers = []
-    database = [database for database in Global.DATABASES if database['id'] == database_id]
+    database = Global.DATABASES.get(database_id)
     if len(database) == 0:
```

Comment:
This should no longer check for len, correct? Instead it should check 'if database is None'?